base/operators.jl

```diff
@@ -525,8 +525,10 @@ julia> identity("Well, what did you expect?")
 """
 identity(x) = x
 
-+(x::Number) = x
-*(x::Number) = x
++(x) = x
+-(x) = Int8(-1)*x
```

Comment:
this seems possibly incorrect for many types (e.g. Unsigned), and less likely for promotion to be defined than negation